docs/src/pages/careers/senior-designer.md

```diff
@@ -52,13 +52,16 @@ That means you'll be working closely with your engineering lead counterpart to s
 As our [design philosophy](https://mui-org.notion.site/Design-philosophy-bdbef60d6fad4134a43870df3e452ebc) states, you'll be conducting and guiding the whole team, making sure everything we do is adding a special design spice that only you can add.
 We imagine a world where design execution is a definitive product differentiator.
 
-### What you'll do on a day-to-day basis
+### What you'll do on your first 3 months
```

Comment:
For the context, the goal of this header is to communicate the responsibility in the role, the goal is to match https://mui-org.notion.site/Designer-ecc6e1c958124a8da07711d57166f6ce, which is public.